src/constants.ts

```diff
@@ -11,6 +11,8 @@ export const EmitterEvents = {
   revalidateFailed: 'revalidateFailed',
 } as const
 
+export type EmitterEvents = (typeof EmitterEvents)[keyof typeof EmitterEvents]
```

Comment:
This adds a bit of ambiguity since the type clashes with the name of the constants map and the type is not strictly necessary to fix the problem.

Do you mind dropping the type declaration here and just keeping the `EmitterEvents` map export you have in the `index.ts` file? That should be enough to fix the problem.

If you want the type available as well, you can instead rename it to the singular form (`EmitterEvent`) and export that alongside the `EmitterEvents` map.

Reply:
Thanks!